### ISSUE TITLE
Long option string breaks the lines, and options render on top of each other

### ISSUE THREAD

**Issue by thomasdarvik** [bvaughn/react-virtualized-select#123]
I am having some problems with long option strings in the component. I am not sure if this is related to `react-select`, or if this is a problem with `react-virtualized-select`.

**How can I prevent text from rendering like this?**
https://i.imgur.com/OmtbeKI.png

As you can see, two options are rendered on top of each other.
This is only happening when the dropdown is small, but it should still be rendered on two lines without breaking, right?

Really nice component. It renders 1600 options without a problem, saving my project :smile:

**Comment by bvaughn** [comment]
You can prevent this with CSS:
```css
white-space: nowrap;
```
Maybe this should even be the default style.

**Comment by thomasdarvik** [comment]
That seems like a good option, however, I see that the text is just continuing out of the dropdown option. It would be great to have the wrap, but prevent the two option lines from overlapping. I will check it out some more.

It might be a good option to have the default style as `nowrap`, but maybe point out that it is default set to `nowrap` in the docs?

**Comment by thomasdarvik** [comment]
**Alternative solution I found**

One solution that fixes my issue, and will render the dropdown option "the correct way" (for me at least), is to add `box-sizing: content-box` and remove the hard-coded `height: 35px` on `.VirtualizedSelectOption` 

``` css
.VirtualizedSelectOption {
    white-space: normal;
    height: auto;
}

.Select, .Select div, .Select input, .Select span {
    box-sizing: content-box !important;
}
```

**Comment by thomasdarvik** [comment]
Possible fix in #124

**Comment by kenzouno1** [comment]
@thomasdarvik i try your css but  not working for me.
